### PR TITLE
Bug 1805994: Remove duplicate entries on operator hub page

### PR DESCRIPTION
**Pull request by jcaianirh**
Fixes bug: https://bugzilla.redhat.com/show_bug.cgi?id=1805450

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx [hunk 1]
```diff
@@ -96,6 +96,8 @@ export const OperatorHubList: React.SFC<OperatorHubListProps> = (props) => {
     },
   );
 
+  const uniqueItems = _.uniqBy(items, 'uid');
```
Comment (benjaminapetersen) [inline frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx]: Strange we have the dups tho?

**Comment by spadgett** [comment]
I believe this is an issue where OLM is not honoring the label selector when listing packagemanifests in all namespaces (see my comment in Bugzilla). But I think we should go ahead with the console change as well to avoid ever having duplicates.

**Comment by spadgett** [comment]
/lgtm cancel

Let's add a `console.warn` if the size is different after `uniqBy` since it shouldn't happen

**Comment by jcaianirh** [comment]
@spadgett added console warning to indicate duplicate packagemanifest from olm query.

**Pull request merged**